docs/source/input_preparation.rst

```diff
@@ -0,0 +1,115 @@
+Instructions for preparing inputs for the OpenFE public benchmark
+#################################################################
+
+Overview & Aims
+***************
+
+This document provides guidance on preparing the `Schrodinger benchmark files <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0/fep_benchmark_inputs/structure_inputs>`_ for use in the OpenFE 2024 public dataset industry benchmark.
+
+Whilst each partner organization will be responsible for preparing inputs using their own tooling, this guidance aims to provide a set of rules to standardize the process and ensure that scientifically equivalent simulation inputs are generated.
+
+This document will be continually updated based on feedback from benchmark partners. Please reach out should you have any questions or require additional information whilst preparing your inputs.
```

Comment:
> reach out

We should have a page for this. Here could you `:ref:` link to a link called "get-in-touch"?